Sourcify.postman_collection.json

```diff
@@ -1,39 +1,819 @@
 {
 	"info": {
-		"_postman_id": "2bc04441-01cb-4ed0-9ea6-3ca30e4666a8",
+		"_postman_id": "c8b9d00d-1e20-46cb-b677-d3da199b3db2",
 		"name": "Sourcify",
 		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
 	},
 	"item": [
 		{
-			"name": "server",
+			"name": "Server",
 			"item": [
 				{
-					"name": "checkByAddress",
-					"request": {
-						"method": "GET",
-						"header": [],
-						"url": {
-							"raw": "{{host}}/checkByAddresses?addresses=0x0000A906D248Cc99FB8CB296C8Ad8C6Df05431c9,0x0A67477639a71bf98528280D3724f465A1814740,0x0A67477639a71bf98528280D3724f465A1814741,0x0a06cc1Ce1105d90ce01752813449A029906aD7b&chainIds=1,3,4,5,42",
-							"host": [
-								"{{host}}"
+					"name": "Verification API 1",
+					"item": [
+						{
+							"name": "checkByAddress",
+							"request": {
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "{{host}}/checkByAddresses?addresses=0x0000A906D248Cc99FB8CB296C8Ad8C6Df05431c9,0x0A67477639a71bf98528280D3724f465A1814740,0x0A67477639a71bf98528280D3724f465A1814741,0x0a06cc1Ce1105d90ce01752813449A029906aD7b&chainIds=1,3,4,5,42",
+									"host": [
+										"{{host}}"
+									],
+									"path": [
+										"checkByAddresses"
+									],
+									"query": [
+										{
+											"key": "addresses",
+											"value": "0x0000A906D248Cc99FB8CB296C8Ad8C6Df05431c9,0x0A67477639a71bf98528280D3724f465A1814740,0x0A67477639a71bf98528280D3724f465A1814741,0x0a06cc1Ce1105d90ce01752813449A029906aD7b"
+										},
+										{
+											"key": "chainIds",
+											"value": "1,3,4,5,42"
+										}
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "verify",
+							"request": {
+								"method": "POST",
+								"header": [],
+								"body": {
+									"mode": "formdata",
+									"formdata": [
+										{
+											"key": "address",
+											"value": "0x0001Db7722Fb4211C24d4aC5E1127353116323d3",
+											"type": "text"
+										},
+										{
+											"key": "chain",
+											"value": "5",
+											"type": "text"
+										},
+										{
+											"key": "files",
+											"type": "file",
+											"src": "/sourcify/repository/contracts/full_match/5/0x0001Db7722Fb4211C24d4aC5E1127353116323d3.zip"
+										}
+									]
+								},
+								"url": {
+									"raw": "{{host}}",
+									"host": [
+										"{{host}}"
+									]
+								}
+							},
+							"response": []
+						}
+					],
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									""
+								]
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									""
+								]
+							}
+						}
+					]
+				},
+				{
+					"name": "Verification API 2",
```

Comment:
I am getting this error for every request inside this collection for APIv2
```json
{
    "error": "Cannot perform the request for the provided Content-Type"
}
```

Reply:
Sorry, I forgot to mention this: All requests that upload files expect them to exist at certain locations on your computer. You can achieve this by setting Sourcify root directory as your Postman working directory. Perhaps we can switch to using Postman variables in the paths of the files being uploaded - and use a Potman pre-request script on every request requiring such variables. Modifying the working directory seemed as a simpler solution to me. Whichever option we choose, it should be documented (something I forgot to do). I can't seem to find a way to add a description to a whole Postman collection, so I'll just add the same description to the three requests including file upload. Also, Postman descriptions don't seem very _visible_ to me, so we can try with something better.